packages/website/pages/docs/guides/deploy-a-contract.mdx

```diff
@@ -96,6 +105,36 @@ This guide will help you deploy a smart contract to Taiko using Foundry or Hardh
 
       ### Deploy your smart contract
       ![deploy your smart contract](/images/guides/deploy-remix/deploy.png)
+
+   </Steps>
+  </Tab> 
+  <Tab>
+   <Steps>
+   ### To deploy your smart contract using deploy, navigate to the root directory of your project and execute the following command:
+
+```bash
+npx thirdweb deploy
+```
+
+Executing this command will trigger the following actions:
+
+- Compiling all the contracts in the current directory.
+- Providing the option to select which contract(s) you wish to deploy.
+- Uploading your contract source code (ABI) to IPFS.
+
+### When it is completed, it will open a dashboard interface to finish filling out the parameters.
+
+- `_name`: contract name
+- `_symbol`: symbol or "ticker"
+- `_royaltyRecipient`: wallet address to receive royalties from secondary sales
+- `_royaltyBps`: basis points (bps) that will be given to the royalty recipient for each secondary sale, e.g. 500 = 5%
+
+### Select `TaikoAlpha2Testnet` as the network
```

Comment:
should this say Alpha3?

Reply:
oh yea, you are right. My bad, fixing it!

Reply:
updated it!